src/routes/ExercisesScreen.tsx

```diff
@@ -29,7 +29,7 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Eleme
   const { params } = route
   const { disciplineTitle } = params
 
-  const load = useCallback(async (params: ExercisesParams) => {
+  const load = useCallback(async (params: ExercisesParams | ExerciseParams) => {
```

Comment:
Line 32 to 38 should be replaced with if you keep the change of the navigation types:
``` ts
  const { data: documents, error, loading, refresh } = useLoadDocuments(discipline)
```

Reply:
Done in https://github.com/digitalfabrik/lunes-app/pull/287/commits/13c7cf9bbed09646973c9eac373532f95d6987e2